massa-consensus-worker/src/state/process_commands.rs

```diff
@@ -33,6 +35,11 @@ impl ConsensusState {
             return Ok(());
         }
 
+        let de_p = DenunciationPrecursor::from(&header);
```

Comment:
Not sure to understand why we add a DenunciationPrecursor here.

Would you mind to explain it and maybe add some doc here?

Reply:
Miss the removal in the controller sorry :-)